src/CSI.IBTA.Administrator/Clients/AuthClient.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Text;
+using CSI.IBTA.Administrator.Interfaces;
+using Newtonsoft.Json;
+
+namespace CSI.IBTA.Administrator.Clients
+{
+    public class AuthClient : IAuthClient
+    {
+        private readonly HttpClient _httpClient;
+        private readonly ILogger<AuthClient> _logger;
+
+        public AuthClient(ILogger<AuthClient> logger, IConfiguration configuration)
+        {
+            _logger = logger;
+            _httpClient = new HttpClient();
```

Comment:
Lets register AuthClient with AddHttpClient extension . HttpClient creation is an expensive process, it requires a bit specific DI registration. https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.Extensions.Http/src/DependencyInjection/HttpClientFactoryServiceCollectionExtensions.cs#L26

More details: https://learn.microsoft.com/en-us/dotnet/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests